src/adaXT/base_model.pyx

```diff
@@ -62,19 +65,19 @@ class BaseModel:
                     self.leaf_builder_class = leaf_builder
                 else:
                     self.leaf_builder_class = LeafBuilderRegression
-            elif tree_type == "LinearRegression":
+            elif tree_type == "Gradient":
                 if predict:
                     self.predict_class = predict
                 else:
-                    self.predict_class = PredictLinearRegression
+                    self.predict_class = PredictLocalPolynomial
                 if criteria:
                     self.criteria_class = criteria
                 else:
-                    self.criteria_class = Linear_regression
+                    self.criteria_class = Partial_quadratic
```

Comment:
The quadratic fits quadratically, no?

Would it not be better suited for the "Gradient" default to make use of the Partial_linear, and then having some other naming for the quadratic?

Reply:
see above